rfcs/0000-rlp-status.md

```diff
@@ -0,0 +1,162 @@
+| Reconciled | True   | "PolicyReconciled"        | "KuadrantPolicy has been successfully reconciled"                           | Yes       |
+|            | False  | "PolicyNotReconciled"     | "KuadrantPolicy failed reconciliation"                                      |           |
+| Enforced   | True   | "PolicyEnforced"          | "KuadrantPolicy has been successfully enforced"                             | Yes       |
+|            | False  | "PolicyPartiallyEnforced" | "KuadrantPolicy has encountered some issues and has been partially applied" |           |
+|            | False  | "PolicyNotEnforced"       | "KuadrantPolicy has encountered an error and can't be applied"              |           |
+
+
+A simplified version and more aligned with the Kubernetes objects implementation could be represented as the following.
+
+**Conditions**
+
+All conditions are top-level.
+
+| Type        | Status | Reason                      | Message                                                                     |
+|-------------|--------|-----------------------------|-----------------------------------------------------------------------------|
+| Progressing | True   | "PolicyCreated"             | "KuadrantPolicy created"                                                    |
+|             | True   | "PolicyUpdated"             | "KuadrantPolicy has been updated"                                           |
+|             | True   | "PolicyApplied"             | "KuadrantPolicy has been successfully applied                               |
+|             | True   | "PolicyReconciled"          | "KuadrantPolicy has been successfully reconciled"                           |
+|             | False  | "PolicyEnforced"            | "KuadrantPolicy has been successfully enforced"                             |
+|             | False  | "PolicyError"               | "KuadrantPolicy has encountered an error"                                   |
+| Enforced    | True   | "PolicyEnforced"            | "KuadrantPolicy has been successfully enforced"                             |
+|             | False  | "PolicyPartiallyEnforced"   | "KuadrantPolicy has encountered some issues and has been partially applied" |
+|             | False  | "PolicyNotEnforced"         | "KuadrantPolicy has encountered an error and can't be applied"              |
+| Failed      | True   | "PolicyValidationError"     | "KuadrantPolicy has failed to validate"                                     |
+|             | True   | "PolicyReconciliationError" | "KuadrantPolicy has encountered a reconciliation error"                     |
+|             | True   | "PolicyServiceError"        | "KuadrantPolicy has encountered has failed to enforce"                      |
+|             | False  | "PolicyEnforced"            | "KuadrantPolicy has been successfully enforced"                             |
```

Comment:
Is the policy being overridden by another policy to be considered a fail? If it's not a fail, then we may need another reason here.